### PR TITLE
bump version 0.3.0 -> 0.3.1 and update CHANGELOG

### PR DESCRIPTION
**Pull request by RCasatta**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 0.3.1 - May 10, 2024
+
+- [Fixed](https://github.com/ElementsProject/elements-miniscript/pull/81) Elip151 hash calculation
```
Comment (delta1) [inline CHANGELOG.md:3]: nit: ELIP is an acronym 

```suggestion
- [Fixed](https://github.com/ElementsProject/elements-miniscript/pull/81) ELIP-151 hash calculation
```

**Review by delta1 — COMMENTED** [review]
utACK 747ebe0b1b9e6ccda07b1d68bd50bee5cdf91299

File: CHANGELOG.md (hunk 1 above)
Comment (RCasatta) [reply CHANGELOG.md]: Changed, thanks.

**Review by apoelstra — APPROVED** [review]
ACK 41d073227f3b636b7942a70dbaae610371fe452d yeah, I think a minor rev is ok because this is a bugfix, even though arguably it is a breaking change

**Comment by apoelstra** [comment]
Tagged and published.

**Pull request merged**